scripts/compare_wandb_configs.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from collections.abc import MutableMapping
+
+import click
+
+from olmo.util import prepare_cli_environment
+
+log = logging.getLogger(__name__)
+
+
+def flatten(dictionary, parent_key="", separator="."):
+    items = []
```

Comment:
Just curious, why not use a dictionary and `dict.update(...)` instead of a list with `extend()`?

Reply:
Because I copy and pasted this from SO 🙈.

Fixed in 049969b22d0b6d0789ae2fc48f2b40111f9ff533.